indico/modules/events/series/blueprint.py

```diff
@@ -0,0 +1,15 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2022 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from indico.modules.events.series.controllers import RHEventSeries
+from indico.web.flask.wrappers import IndicoBlueprint
+
+
+_bp = IndicoBlueprint('event_series', __name__, url_prefix='/series/')
```

Comment:
```suggestion
_bp = IndicoBlueprint('event_series', __name__, url_prefix='/event-series')
```

I think that name in the URL is a bit better, and there shouldn't be a trailing slash the the prefix is really just prepended to the url rules below - so it'd generate `/series//`